github-actions/run-github-release/README.md

```diff
@@ -0,0 +1,42 @@
+# run-github-release
+
+The `run-github-release` action allows you to run any binary release on GitHub.
+See [`actions.yml`] for configuration options.
+
+[`actions.yml`]: ./actions.yml
+
+## Example
+This example builds an mdbook project on linux.
+
+```yaml
+- uses: rust-lang/simpleinfra/github-actions/run-github-release@master
```

Comment:
It's probably better to pin this to a specific version so that future modifications won't break your existing users:

```suggestion
- uses: rust-lang/simpleinfra/github-actions/run-github-release@v1
```

Of course, there will be a need to create a new release in this repository, but it's better to be safe than sorry!